gems/pending/spec/postgres_ha_admin/failover_databases_spec.rb

```diff
@@ -1,5 +1,9 @@
-describe FailoverDatabases do
+require 'postgres_ha_admin/failover_databases'
+
+describe PostgresHaAdmin::FailoverDatabases do
   before do
+    @failover_databases = described_class.new(Rails.root)
```

Comment:
You can do this with `subject` if you want.
`subject { described_class.new(Rails.root) }` then reference it as `subject.all_databases` ... etc